frontend/packages/topology/src/components/GraphComponent.tsx

```diff
@@ -55,11 +56,11 @@ const GraphComponent: React.FC<GraphComponentProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [layout]);
 
+  const refs = useCombineRefs(dndDropRef, panZoomRef);
   const bounds = element.getBounds();
   return (
     <>
       <rect
-        ref={dndDropRef}
```

Comment:
This needs to be on this rect otherwise.
See PR https://github.com/openshift/console/pull/4063